src/sync_template.sh

```diff
@@ -493,6 +510,12 @@ function arr_commit() {
 
   echo "::group::commit changes"
 
+  # Only commit if merge commit was not already created by git pull
+  if git log -1 --pretty=%B | grep -q "Merge"; then
```

Comment:
Hi @cjlpowers sorry for my late comment.
Actually I see a small issue with this part. If I understand right it requires that `Merge` is part of the commit message.
Actually it is not always the case for merge commits.

Possibly there is a better way to find out if it is a merge commit.